helmfile/charts/user-rbac/templates/rolebindings/workload-admin.yaml

```diff
@@ -20,8 +20,9 @@ subjects:
   kind: Group
   name: {{ $group }}
 {{- end }}
+{{- $known := lookup "v1" "Service" "default" "kubernetes" }}
 {{- $value := lookup "rbac.authorization.k8s.io/v1" "RoleBinding" $namespace "extra-workload-admins" }}
-{{- if and (or $.Release.IsInstall $.Release.IsUpgrade) (not $value) }}
+{{- if or $.Release.IsInstall $.Release.IsUpgrade | and $known (not $value) }}
```

Comment:
How does the `|` operator work in this situation? Like does the `and` function have input?

Reply:
Well let me tell you cause I've finally read up on how the pipe operator works in helm :sweat_smile:

When you use a pipe the result of the expression will be used as the **last** argument for the next expression so this one would be equivalent to `and $known (not $value) (or $.Release.IsInstall $.Release.IsUpgrade)`. And this applies to all functions.

Reply:
Righ, yea that kinda makes sense. But would it maybe be a bit easier to read if we changed it to the equivalent expression in that case? At least for me that's more readable.